src/main/java/seedu/address/logic/commands/Command.java

```diff
@@ -16,21 +16,25 @@
     /**
      * Executes the command and returns the result message.
      *
-     * @param model {@code Model} which the command should operate on.
-     * @return feedback message of the operation result for display
+     * @param model {@code Model} which contains the model which the command operates on
      * @throws CommandException If an error occurs during command execution.
      */
-    public abstract CommandResult execute(Model model) throws CommandException;
-
-    protected void requireDeliveryModel(Model model) throws CommandException {
-        if (!(model instanceof DeliveryModel)) {
-            throw new CommandException(MESSAGE_INCORRECT_MODEL);
+    public CommandResult execute(Model model) throws CommandException {
+        if (model instanceof InventoryModel) {
```

Comment:
Can just check why are the types opposite?

Reply:
What do you mean

Reply:
Ah nvm my mistake, understood whats happening alrd